README.md

```diff
@@ -55,6 +55,15 @@ npm run build
 npm run preview --port 4173
 ```
 
+To update GitHub and npmjs stats for packages, use:
+
+```bash
+npm run fetch-data
+```
+
+When adding a package to .data.json file, skip `github` and `nodejs` fields, and
+then run `fetch-data` script.
```

Comment:
@rmjordas i think, during github builds we will not access external API's

Instead, when updating page locally - we have option to run `fetch-data` for loading package stats.

Maybe, next step would be running this regularly as part of CI, but for now let's start with less risky approach.

Github allows 30 search requests per minute, and octokit library handles automatic retries.
For npm api i haven't found any details on API limits, but probably we can figure it out when we convert more pages.

Reply:
I could imagine a GitHub Action to handle regular updates for the stats. And I've peeked a bit at the code and at the deploy preview and I think it looks neat. Good job 👍 I've also added you to the repository so you can work with the `next` branch however you like.

Reply:
I was thinking about splitting the data - one json file - storing the links/descriptions, and another - with fetched stats, but decided against it, single file is simpler to work with from programming point of view.

So, we just add "name", "url" and "description" into json file - and run `fetch-data` script, it populates other fields.

Right now it will refetch stats for everything, but i can upgrade it to fetch only new data when asked.